src/less/components/payex/togglebox.less

```diff
@@ -0,0 +1,44 @@
+@import "../togglebox.less";
+
+/* For theme-white */
+.togglebox:has(input.theme-white) input[type="checkbox"]:not(:disabled) + label:after {
+    background-color: var(--brand-secondary);
+}
+
+.togglebox:has(input.theme-white) input[type="checkbox"]:not(:disabled) + label:before {
+    background-color: var(--white);
+}
+
+.togglebox:has(input.theme-white) input[type="checkbox"]:not(:disabled):checked + label:before {
+    background-color: var(--brand-success);
+}
+
+.togglebox:has(input.theme-white) input[type="checkbox"]:not(:disabled):checked + label:after {
+    background-color: var(--white);
+}
```

Comment:
@PriJoh you can make it easier by re-addressing the togglebox custom-properties. This way no need to going through all the selectors again.

e.g.
```
.togglebox:has(input.theme-white) {
    --checked-thumb-color: var(--white);
    --checked-track-color: var(--brand-success);
    --unchecked-thumb-color: var(--white);
    --unchecked-track-color: var(--brand-secondary);
}
```

and doing the same for the gray theme

Reply:
I have looked at all the comments and they should now be resolved :)